README.md

```diff
@@ -238,7 +220,69 @@ the [Maven version range format](https://cwiki.apache.org/confluence/display/MAV
 | (,1.0],[1.2,) | x <= 1.0 or x >= 1.2. Multiple sets are comma-separated                       |
 | (,1.1),(1.1,) | This excludes 1.1 if it is known not to work in combination with this library |
 
-#### External Dependencies: Runs
+#### Local Files
+
+You can also include files built by other tasks in your project, for example, jar tasks of other source sets.
+
+When wanting to build a secondary jar for a coremod or service, you could define a separate source set `service`,
+add a jar task to package it and then include the output of that jar like this:
+
+```groovy
+sourceSets {
+    service
+}
+
+
+neoForge {
+    // ...
+    mods {
+        // ...
+        // To make the service load in dev
+        'service' {
+            sourceSet sourceSets.service
+        }
+    }
+}
+
+def serviceJar = tasks.register("serviceJar", Jar) {
```

Comment:
First of all, duplicate manifest stuff. Second of all, this whole example feels like it would be a better place to use a feature variant. A source set with its own jar that represents some extra feature of a module is exactly what features are meant to represent. See https://docs.gradle.org/current/userguide/feature_variants.html and the docs for FeatureSpec. Am I missing something obvious here?

Reply:
I feel like source sets will be more familiar to most modders?